bucket/jim.json

```
@@ -0,0 +1,13 @@
+{
+    "version": "0.9.9.9.7",
+    "description": "TF2 Item Manager",
+    "url": "http://www.jengerer.com/item_manager/downloads/item_manager.zip",
+    "homepage": "http://www.jengerer.com/item_manager/",
```

Add a license field:

```suggestion
    "homepage": "http://www.jengerer.com/item_manager/",
    "license": "Proprietary",
```